lightning/src/ln/channel.rs

```diff
@@ -585,69 +587,72 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		let mut local_config = (*config).channel_options.clone();
 
 		if config.own_channel_config.our_to_self_delay < BREAKDOWN_TIMEOUT {
-			return Err(ChannelError::Close("Configured with an unreasonable our_to_self_delay putting user funds at risks"));
+			return Err(ChannelError::Close(format!("Configured with an unreasonable our_to_self_delay ({}) putting user funds at risks", config.own_channel_config.our_to_self_delay)));
 		}
 
 		// Check sanity of message fields:
 		if msg.funding_satoshis >= MAX_FUNDING_SATOSHIS {
-			return Err(ChannelError::Close("funding value > 2^24"));
+			return Err(ChannelError::Close(format!("funding must be smaller than {}. It was {}", msg.funding_satoshis, MAX_FUNDING_SATOSHIS)));
 		}
 		if msg.channel_reserve_satoshis > msg.funding_satoshis {
-			return Err(ChannelError::Close("Bogus channel_reserve_satoshis"));
+			return Err(ChannelError::Close(format!("Bogus channel_reserve_satoshis({}). must be not greater than funding_satoshis: {}", msg.channel_reserve_satoshis, msg.funding_satoshis)));
```

Comment:
nit: `s/Bogus channel_reserve_satoshis({})/Bogus channel_reserve_satoshis ({})/`
and `s/must/Must` and in a few other places I think (capitalize the first letter of a sentence + space before open parentheses)